source/expectations/data_values/ut_compound_data_helper.pkb

```diff
@@ -211,9 +208,9 @@ create or replace package body ut_compound_data_helper is
                  r.duplicate_no,
                  ucd.column_value.getclobval() col_val,
                  ucd.column_value.getRootElement() col_name,
-                 ut_compound_data_helper.get_pk_value(:join_xpath,r.item_data) pk_value
+                 r.pk_value
                  from 
-                  (select ]'||l_column_filter||q'[, ucd.item_no, ucd.item_hash, i.pk_hash, i.duplicate_no
+                  (select ]'||l_column_filter||q'[, ucd.item_no, ucd.item_hash, i.pk_hash, i.pk_value, i.duplicate_no
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An illegal character with code point 10 was found in this literal. ([plsql:LiteralsNonPrintableCharactersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ALiteralsNonPrintableCharactersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=760&issues=AWY_N4w2nzfF1D4JsH2N&open=AWY_N4w2nzfF1D4JsH2N)

[ISSUE_KEY]: AWY_N4w2nzfF1D4JsH2N